package.json

```diff
@@ -405,12 +405,12 @@
 			"view/title": [
 				{
 					"command": "intuita.caseBreakdown",
-					"when": "view == intuitaMainWebviewExplorer || view == intuitaMainWebview",
+					"when": "false",
```

Comment:
hang on, then this command will be invisible right?

Reply:
yes, to my understanding, we don't need this command unless we provide options between caseBreakdown vs folderBreakdown.